lib/src/barback/dart2js_transformer.dart

```diff
@@ -225,36 +225,16 @@ class _BarbackCompilerProvider implements dart.CompilerProvider {
   /// The map of previously loaded files.
   ///
   /// Used to show where an error occurred in a source file.
-  final _sourceFiles = new Map<String, SourceFile>();
+  final _sourceFiles = <String, SourceFile>{};
 
-  // TODO(rnystrom): Make these configurable.
-  /// Whether or not warnings should be logged.
-  var _showWarnings = true;
-
-  /// Whether or not hints should be logged.
-  var _showHints = true;
-
-  /// Whether or not verbose info messages should be logged.
-  var _verbose = false;
-
-  /// Whether an exception should be thrown on an error to stop compilation.
-  final _throwOnError = false;
-
-  /// This gets set after a fatal error is reported to quash any subsequent
-  /// errors.
-  var _isAborting = false;
+  _BarbackCompilerInput inputProvider;
```

Comment:
Can you make these final?

Reply:
I can't because they need to share the _sourceFiles map